app/models/agents/http_status_agent.rb

```diff
@@ -12,6 +12,7 @@ class HttpStatusAgent < Agent
 
     form_configurable :url
     form_configurable :disable_redirect_follow, type: :array, values: ['true', 'false']
+    form_configurable :changes_only, type: :array, values: ['true', 'false']
```

Comment:
@dsander 

That sounds fine to me, I can make that change.

But what about the line above this change?  I've already used the true/false array once in this agent.  Perhaps I should convert that, if possible? (I don't know if it is, I don't know how the boolean would make the strings 'true' and 'false'.)

I wish I had gone with ```:boolean``` to start, but I think there's something for **consistency.**  It would bug me to see one conditional with a true/false dropdown, then a checkbox for the next conditional.